### PR TITLE
Various series updated.

### PR DESCRIPTION
**Pull request by OxyMauron**
Misc series changes, mostly special matched.

| AniDB | TVDB/TMDB/IMDB | Notes |
| --- | --- | --- |
| https://anidb.net/anime/2504 | https://thetvdb.com/index.php?tab=series&id=160621 | Season 1 |
| https://anidb.net/anime/2546 | https://thetvdb.com/index.php?tab=series&id=363417 | Season 1 |
| https://anidb.net/anime/2547 | https://thetvdb.com/index.php?tab=series&id=405844 | Season 1 |
| https://anidb.net/anime/2549 | https://thetvdb.com/index.php?tab=series&id=405890 | Season 1 |
| https://anidb.net/anime/3461 | https://thetvdb.com/index.php?tab=series&id=111011 | Corrected episode offset |
| https://anidb.net/anime/5376 | https://thetvdb.com/index.php?tab=series&id=232991 | Special mapping |
| https://anidb.net/anime/7963 | https://thetvdb.com/index.php?tab=series&id=19243 | Special mapping |
| https://anidb.net/anime/9833 | https://thetvdb.com/index.php?tab=series&id=74504 | Special mapping |
| https://anidb.net/anime/11351 | N/A | labeled as hentai |
| https://anidb.net/anime/11852 | https://thetvdb.com/index.php?tab=series&id=71863 | Special mapping |
| https://anidb.net/anime/12398 | https://thetvdb.com/index.php?tab=series&id=275609 | Special mapping |
| https://anidb.net/anime/12701 | https://thetvdb.com/index.php?tab=series&id=81110 | Season 3 |
| https://anidb.net/anime/12919 | https://thetvdb.com/index.php?tab=series&id=26796 | Special mapping |
| https://anidb.net/anime/13006 | https://thetvdb.com/index.php?tab=series&id=321820 | Special mapping |
| https://anidb.net/anime/13269 | https://thetvdb.com/index.php?tab=series&id=111011 | Special mapping |
| https://anidb.net/anime/13447 | https://thetvdb.com/index.php?tab=series&id=379282 | Season 1 |
| https://anidb.net/anime/13526 | https://thetvdb.com/index.php?tab=series&id=340261 | Season 1 |
| https://anidb.net/anime/13591 | https://thetvdb.com/index.php?tab=series&id=197261 | Special mapping |
| https://anidb.net/anime/13688 | https://thetvdb.com/index.php?tab=series&id=340261 | Special mapping |
| https://anidb.net/anime/13926 | https://thetvdb.com/index.php?tab=series&id=340261 | Special mapping |
| https://anidb.net/anime/14501 | https://thetvdb.com/index.php?tab=series&id=71863 | Special mapping |
| https://anidb.net/anime/14506 | https://thetvdb.com/index.php?tab=series&id=76665 | Special mapping |
| https://anidb.net/anime/14616 | https://thetvdb.com/index.php?tab=series&id=71863 | Special mapping |
| https://anidb.net/anime/15297 | https://thetvdb.com/index.php?tab=series&id=388485 | Season 1 |
| https://anidb.net/anime/16008 | https://thetvdb.com/index.php?tab=series&id=392944 | Special mapping |
| https://anidb.net/anime/16047 | N/A | labeled as hentai |
| https://anidb.net/anime/17634 | https://thetvdb.com/index.php?tab=series&id=71863 | Special mapping |
| https://anidb.net/anime/18159 | https://thetvdb.com/index.php?tab=series&id=379282 | Season 2 |

### CODE REVIEW

File: anime-list-master.xml [hunk 1]
```diff
@@ -39938,8 +39938,11 @@
   <anime anidbid="14505" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>TV Yarou Nanana</name>
   </anime>
-  <anime anidbid="14506" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="14506" tvdbid="76665" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
     <name>Yuu Yuu Hakusho Kanzen Shinsaku Animation</name>
+	<mapping-list>
+      <mapping anidbseason="1" tvdbseason="0">;1-17+18;</mapping>
+    </mapping-list>
```
Comment (BrutuZ) [inline anime-list-master.xml:39945]: Opening is over-indented

Comment (OxyMauron) [reply anime-list-master.xml]: Yeah, that was the issue. Fixed.

File: anime-list-master.xml [hunk 2]
```diff
@@ -48522,7 +48525,7 @@
   <anime anidbid="17633" tvdbid="397280" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Tokyo Mew Mew New (2023)</name>
   </anime>
-  <anime anidbid="17634" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="17634" tvdbid="" defaulttvdbseason="0" episodeoffset="51" tmdbid="" imdbid="">
```
Comment (BrutuZ) [inline anime-list-master.xml:48528]: Missing the TVDBID

Comment (OxyMauron) [reply anime-list-master.xml:48528]: Also fixed.

**Comment by BrutuZ** [comment]
That's a lot of entries to cross-check. Since they're all new mappings I'll just blindly trust the IDs match 🙈

**Pull request merged**